EcoSoapBank/Model Controller/ImpactController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  PickupController.swift
+//  EcoSoapBank
```

Comment:
Definitely not very relevant, but it looks like this was copied and the headers not adjusted.